src/main/java/org/generationcp/middleware/domain/genotype/SampleGenotypeSearchRequestDTO.java

```diff
@@ -0,0 +1,181 @@
+package org.generationcp.middleware.domain.genotype;
+
+import org.generationcp.middleware.domain.etl.MeasurementVariable;
+import org.generationcp.middleware.domain.search_request.SearchRequestDto;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@AutoProperty
+public class SampleGenotypeSearchRequestDTO extends SearchRequestDto {
+
+	private int studyId;
+
+	private List<MeasurementVariable> sampleGenotypeVariables;
+
+	private List<Integer> takenByIds;
+
+	/**
+	 * List of columns that will be included and retrieved in the query.
+	 * If no columns are specified, the query will retrieve all columns by default.
+	 */
+	private Set<String> visibleColumns = new HashSet<>();
+
+	/**
+	 * This is used by DataTables to ensure that the Ajax returns from server-side processing requests are drawn in sequence
+	 */
+	private String draw;
```

Comment:
is this still required? I remember we stopped using this in some tables so our services are not designed for an specific frontend solution